console/service_provider.go

```diff
@@ -17,7 +17,7 @@ func (receiver *ServiceProvider) Register(app foundation.Application) {
 		name := "artisan"
 		usage := "Goravel Framework"
 		usageText := "artisan [global options] command [options] [arguments...]"
-		return NewApplication(name, usage, usageText, app.Version()), nil
+		return NewApplication(name, usage, usageText, app.Version(), true), nil
```

Comment:
Should set `true` manually, otherwise, `go run .` will be the same as `go run . artisan`.